backup/setup/systems.json

```diff
@@ -19,18 +19,34 @@
 			"$oid": "0000d186816abba584714c92"
 		},
 		"alias": "Schulserver",
-		"type": "oauthIserv",
 		"__v": 0,
+		"ldapConfig" : {
+			"active" : true,
+			"federalState": {
+				"$oid": "0000b186816abba584714c58"
+			},
+			"url" : "ldap://sc-openldap-svc.sc-common.svc.cluster.local:389",
+			"rootPath" : "dc=de,dc=example,dc=org",
+			"searchUser" : "cn=admin,dc=example,dc=org",
+			"searchUserPassword" : "${SEARCH_USER_PASSWORD}",
+			"provider" : "iserv-idm",
+			"providerOptions" : {
+				"userAttributeNameMapping" : {},
+				"roleAttributeNameMapping" : {},
+				"classAttributeNameMapping" : {}
+			}
+		},
+		"type": "ldap",
 		"oauthConfig": {
-			"clientId": "SCHULSERVER_CLIENT_ID",
-			"clientSecret": "SCHULSERVER_CLIENT_SECRET",
+			"clientId": "${ISERV_CLIENT_ID}",
```

Comment:
During export secrets are replace by a patter e.g.  `\${${system.alias!.toLocaleUpperCase()}_CLIENT_SECRET}`. Thus this needs to stay `SCHULSERVER_CLIENT_ID` otherwise it will be overwritten in the next export.

Reply:
DO you know when the export is triggered? I don't see an obvious way of linking the placeholder values with the secrets in 1Password. Assuming a naming schema (derived) from alias may work in some cases, however can not be a general solution. We may add warnings to the 'resolvePlaceholder' function, if a placeholder could not be resolved. What's your thoughts?

Reply:
I see, we could extend the export mechanism and make it fetch the JSON content in order to set the original placeholders then. Does that make sense to you?